interfaces/mount/entry.go

```diff
@@ -72,12 +73,16 @@ func (a *Entry) Equal(b *Entry) bool {
 //  tab       => (\011)
 //  newline   => (\012)
 //  backslash => (\134)
-func escape(s string) string {
-	return whitespaceReplacer.Replace(s)
-}
+var escape = strings.NewReplacer(" ", `\040`, "\t", `\011`, "\n", `\012`, "\\", `\134`).Replace
```

Comment:
Instead of documenting this and then repeating it with code hoping we have both in sync, it'd be better to format the code itself in a way that may be read easily:
```
var escape = strings.NewReplacer(
        " ", `\040`,
        "\t", `\011`,
        "\n", `\012`,
        "\\", `\134`,
)
```

That sort of issue shows up very often. Sometimes that's not feasible, but when it is, code that speaks for itself is much better than comments.